packages/web-components/src/components/ic-text-field/ic-text-field.tsx

```diff
@@ -45,6 +45,35 @@ import {
 let inputIds = 0;
 const MUTABLE_ATTRIBUTES = [...IC_INHERITED_ARIA, "title"];
 
+const ignoredKeyboardCharacters = [
```

Comment:
immutable global const should be all uppercase

> Reason
- https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/const#basic_const_usage

> Optional reading

- https://stackoverflow.com/questions/838929/naming-why-should-named-constants-be-all-uppercase-in-c-java
- https://akshayvjain.medium.com/uppercase-with-is-pretty-much-the-standard-for-hard-coded-constants-in-javascript-67a9bd374793